root/user/UserProfile.js

```diff
@@ -560,12 +627,44 @@ const UserProfileStatistics = ({
           </tr>
         </tfoot>
       </table>
+
+      <table
+        className="statistics"
+        title={l('This table shows a summary ' +
+                 'of entities added by this editor.')}
+      >
+        <thead>
+          <tr>
+            <th colSpan="2">
+              {exp.l('Added entities')}
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {Object.entries(ADDED_ENTITIES_TYPES)
+          .sort((a, b) => a[1] > b[1])
```

Comment:
Following the above suggestion, `l` should be called here instead, just before `sort`.